crates/bevy_feathers/src/colors.rs

```diff
@@ -0,0 +1,28 @@
+use bevy_color::Color;
+
+/// Black
+pub const BLACK: Color = Color::oklcha(0.0, 0.0, 0.0, 1.0);
+/// Gray 0
+pub const GRAY_0: Color = Color::oklcha(0.2414, 0.0095, 285.67, 1.0);
+/// Gray 1
+pub const GRAY_1: Color = Color::oklcha(0.2866, 0.0072, 285.93, 1.0);
+/// Gray 2
+pub const GRAY_2: Color = Color::oklcha(0.3373, 0.0071, 274.77, 1.0);
+/// Gray 3
+pub const GRAY_3: Color = Color::oklcha(0.3992, 0.0101, 278.38, 1.0);
+/// Warm Gray 3
+pub const WARM_GRAY_1: Color = Color::oklcha(0.3757, 0.0017, 286.32, 1.0);
+/// Light Gray 1
+pub const LIGHT_GRAY_1: Color = Color::oklcha(0.7607, 0.0014, 286.37, 1.0);
+/// Light Gray 2
+pub const LIGHT_GRAY_2: Color = Color::oklcha(0.6106, 0.003, 286.31, 1.0);
+/// White
+pub const WHITE: Color = Color::oklcha(1.0, 0.000000059604645, 90.0, 1.0);
+/// Accent
```

Comment:
Describing the basic color here, like you did for XYZ, would be helpful.

Reply:
I've removed the descriptions in favor of swatches.

BTW, I didn't get the names from the basic colors. I used the color picker app on my phone, but that's not an exact process (depending on illumination).